src/test/java/com/wikia/webdriver/elements/mercury/components/Header.java

```diff
@@ -0,0 +1,82 @@
+package com.wikia.webdriver.elements.mercury.components;
+
+import com.wikia.webdriver.common.contentpatterns.MercuryMessages;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.elemnt.Wait;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class Header {
+
+  @FindBy(css = ".wiki-page-header__title")
+  private WebElement pageTitle;
+
+  @FindBy(css = ".wiki-page-header .has-hero-image")
+  private WebElement headerWithHeroImage;
+
+  @FindBy(css = ".wiki-page-header:not(.has-hero-image)")
+  private WebElement headerWithoutHeroImage;
+
+  private Wait wait;
+
+  public Header(WebDriver driver) {
+    this.wait = new Wait(driver);
+
+    PageFactory.initElements(driver, this);
+  }
+
+  public Header isPageTitleVisible() {
+    wait.forElementVisible(pageTitle);
+    PageObjectLogging.log("Page header title", MercuryMessages.VISIBLE_MSG, true);
+
+    return this;
+  }
+
+  public Header isHeroImageVisible() {
+    wait.forElementVisible(headerWithHeroImage);
+    PageObjectLogging.log("Hero image", MercuryMessages.VISIBLE_MSG, true);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Hero image" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)